test/jasmine/tests/gl_plot_interact_test.js

```diff
@@ -309,8 +309,8 @@ describe('Test gl3d plots', function() {
         .then(function() {
             expect(countCanvases()).toEqual(1);
             expect(gd.layout.scene).toEqual(sceneLayout);
-            expect(gd.layout.xaxis).toBeUndefined();
-            expect(gd.layout.yaxis).toBeUndefined();
+            expect(gd.layout.xaxis === undefined).toBe(true);
+            expect(gd.layout.yaxis === undefined).toBe(true);
```

Comment:
Something is off with `toBeUndefined`?

Reply:
when you `expect(somePotentiallyBigObject).toBeUndefined()`, a failure swamps you with the whole stringified object. The killer was `expect(scene)` whose string representation was so big it took me forever to find the tracebacks. If the object you might get on failure is small, `toBeUndefined` is still a good idea as it shows you what you got.